pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -1438,6 +1439,15 @@ func TestDeploymentConfigChanged(t *testing.T) {
 			},
 			expect: true,
 		},
+		{
```

Comment:
Should we add a test case for detecting the changing of `deployment.Spec.Template.Spec.Containers[0].LivenessProbe.terminationGracePeriodSeconds`? Or maybe it could be added to one of the test cases above?

Reply:
I added a test case for it.